src/test/kotlin/RouterTest.kt

```diff
@@ -3,33 +3,64 @@ import router.Router
 import kotlin.test.*
 
 class RouterTest {
-    @Test fun testExpectAddRouteMemberToAddSimpleGetToRoutes() {
-        val badRequestController = BadRequestController()
-        val notFoundController = NotFoundController()
-        val routes = Router(badRequestController, notFoundController)
+    private val badRequestController = BadRequestController()
+    private val notFoundController = NotFoundController()
+    private val router = Router(badRequestController, notFoundController)
+
+    @Test
+    fun `expect addRoute member to add GET method to the simple_get route`() {
         val simpleGetController: Controller = SimpleGetController()
-        routes.addRoute("GET", "/simple_get", simpleGetController)
-        val expectation: MutableMap<String, MutableMap<String, Controller>> = mutableMapOf("GET" to mutableMapOf("/simple_get" to simpleGetController))
-        assertEquals(expectation, routes.routes)
+        router.addRoute("GET", "/simple_get", simpleGetController)
+        val expectation: MutableMap<String, MutableMap<String, Controller>> = mutableMapOf("/simple_get" to mutableMapOf("GET" to simpleGetController))
+        assertEquals(expectation, router.routes)
+    }
+
+    @Test
+    fun `expect addRoute member to add GET method to simple_get_with_body route`() {
+        val simpleGetWithBodyController: Controller = SimpleGetWithBodyController()
+        router.addRoute("GET", "/simple_get_with_body", simpleGetWithBodyController)
+        val expectation: MutableMap<String, MutableMap<String, Controller>> = mutableMapOf("/simple_get_with_body" to mutableMapOf("GET" to simpleGetWithBodyController))
+        assertEquals(expectation, router.routes)
     }
 
-    @Test fun testExpectAddRouteMemberToAddSimpleGetBodyToRoutes() {
-        val badRequestController = BadRequestController()
-        val notFoundController = NotFoundController()
-        val routes = Router(badRequestController, notFoundController)
+    @Test
+    fun `expect getController to return a controller from simple_get_with_body route`() {
         val simpleGetWithBodyController: Controller = SimpleGetWithBodyController()
-        routes.addRoute("GET", "/simple_get_with_body", simpleGetWithBodyController)
-        val expectation: MutableMap<String, MutableMap<String, Controller>> = mutableMapOf("GET" to mutableMapOf("/simple_get_with_body" to simpleGetWithBodyController))
-        assertEquals(expectation, routes.routes)
+        router.addRoute("GET", "/simple_get_with_body", simpleGetWithBodyController)
+        val controller = router.getController("GET", "/simple_get_with_body")
+        assertIs<Controller>(controller)
```

Comment:
Should the assertion be changed to this? Similar question for other similar assertions.
```suggestion
        assertIs<SimpleGetWithBodyController>(controller)
```

Reply:
Yes, thank you for pointing this out!